sos/collector/clusters/ocp.py

```diff
@@ -108,8 +109,9 @@ def _attempt_oc_login(self):
         token
         """
         _res = self.exec_primary_cmd(
-            self.fmt_oc_cmd("login --insecure-skip-tls-verify=True --token=%s"
-                            % self.token)
+            self.fmt_oc_cmd("login --insecure-skip-tls-verify=True "
+                            f"--token={self.token} "
+                            f"{self.get_option('api-url')}")
```

Comment:
This doesn't make sense to inject when the default value of `api-url` is used. We'd either break things, or introduce ambiguity into the login command, by passing a spurious `False` value at the end of the string.

Reply:
@alosadagrande checked and it still works when not specifying it. But, would it be cleaner to move from False to '' ?

Reply:
For a project that has a _ton_ of external calls, it is imperative that sos be as exact and semantically correct as possible.

Reply:
How it could work with default `False`? Per my testing, `collector`'s `get_option` should return `bool` typed `False`, hence `fmt_oc_cmd` should get as an argument a string like

    login --insecure-skip-tls-verify=True --token=..... False

Reply:
As the default was changed to empty string, this comment is resolved.